logger/logger.go

```diff
@@ -74,6 +75,12 @@ type logger struct {
 	forcedDebug     bool
 	loggerMu        sync.Mutex
 	timestampFormat string
+	tags            []LogTag
+}
+
+type LogTag struct {
+	Name     string   `json:"name"`
+	LogLevel LogLevel `json:"log_level"`
```

Comment:
`LogLevel` seems redundant here, maybe just `Level`?

Reply:
As the main entity here is "Tag" i think it might be misleading. It is the log level, and not the level of the tag.